dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -3085,6 +3085,51 @@
           }
         }
       }
+    },
+    "secondaryAttachment": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "name",
+          "attachmentId"
+        ],
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "confirmationCode": {
+            "type": "string"
+          },
+          "attachmentId": {
+            "type": "string",
+            "enum": [
+              "L229",
+              "L018",
+              "L034",
+              "L048",
+              "L049",
+              "L029",
+              "L034",
+              "L034",
+              "L023",
+              "L015"
+            ],
+            "enumNames": [
```

Comment:
Not sure what the requirements are for this but it might be good to alphabetize this list.

Reply:
Hi, @rianfowler : I asked and the requirements are to leave as is and not in alphabetical order. Thank you!